docs/reference/sql/functions/conditional.asciidoc

```diff
@@ -98,6 +98,31 @@ an error message would be returned, mentioning that *'foo'* is of data type *key
 which does not match the expected data type *integer* (based on result *10*).
 ===============================
 
+[[sql-functions-conditional-case-groupby-custom-buckets]]
+[TIP]
```

Comment:
I would remove the TIP header and promote this to a section (`=====`) - I think it's big enough to warrant a proper entry.